product_dimension/product.py

```diff
@@ -21,7 +21,7 @@
 
 
 class Product(models.Model):
-    _inherit = 'product.product'
+    _inherit = 'product.template'
```

Comment:
ah, then nothing I did should be kept...I don't need it there so I'll just keep mine for now.